iroh/src/magicsock/remote_map/remote_state/path_state.rs

```diff
@@ -54,7 +54,16 @@ impl RemotePathState {
     /// Insert a new address of an open path into our list of paths.
     ///
     /// This will emit pending resolve requests and trigger pruning paths.
-    pub(super) fn insert_open_path(&mut self, addr: transports::Addr, source: Source) {
+    pub(super) fn insert_open_path(
+        &mut self,
+        addr: transports::Addr,
+        source: Source,
+        metrics: &MagicsockMetrics,
+    ) {
+        match addr {
+            transports::Addr::Ip(_) => metrics.transport_paths_ip_added.inc(),
```

Comment:
Names are pretty subjective, I think I'll argue for `transport_{ip|relay}_paths_added`. Reason being that `paths_added` is the counter unit and `tranport_ip` belongs together as `ip` is the qualifier to transports. Good call to put transports at the front though.

Now what I *really* want is labels so we can do `paths_added{transport=ip}` but hey.